batch_import/camio_hooks.py

```diff
@@ -86,14 +136,16 @@ def dateshift(timestamp, seconds, format = "%Y-%m-%dT%H:%M:%S.%f"):
     date = date + datetime.timedelta(seconds=seconds)
     return date.strftime(format)
 
-def get_device_id():
-    if not CAMIO_PARAMS.get('device_id'):
+def get_device_id(fail=True):
+    """ if fail we exit if the ID cannot be located """
+    if CAMIO_PARAMS.get('device_id') is None:
         device = os.environ.get(CAMIO_BOX_DEVICE_ID_ENVVAR)
```

Comment:
just to convey that this is the string `device_id`, prefer that we consistently use `device_id` when referring to the string id only (vs. a device object)